pkg/virt-operator/application.go

```diff
@@ -123,6 +123,18 @@ func Execute() {
 		DaemonSetCache:          app.informerFactory.OperatorDaemonSet().GetStore(),
 	}
 
+	onOpenShift, err := util.IsOnOpenshift(app.clientSet)
+	if err != nil {
+		golog.Fatalf("Error determining cluster type: %v", err)
+	}
+	if onOpenShift {
```

Comment:
What would you think about using a fake informer in the event that this isn't openshift?  That would prevent us from ever having to worry about accidently trying to access the SCCCache when it's nil.

I did something like that for the DataVolume informer in pkg/virt-controller/watch/application.go

Reply:
yes, very good idea. I didn'like the nil check, but didn't think of a fake informer. Thanks and done :+1: